### DISCUSSION TITLE
Connection issue

### DISCUSSION THREAD

**Discussion by meDariatka** [NightscoutFoundation/xDrip#3033] [General]
Hi, I am new Dexcom G7 user and having connection problem to app on my mobile one+ 6. It connected and took 1 reading and stopped. Please see attached pic of with what I believe is causing an issue.
![Screenshot_20230821-125324](https://github.com/NightscoutFoundation/xDrip/assets/142795246/86df010c-a687-496b-b8a1-50390e14e8ba)

**Comment by Navid200** [comment]
No, that is not causing the issue.  That page has to with uploading to Nightscout.

Please post a screenshot of the G5/G6/G7 status page.

**Comment by meDariatka** [comment]
Hi, thanks for your quick response. Please see attached screenshot. Thanks
a lot. Daria

**Comment by meDariatka** [comment]
Hi David,

I have no clue what you did but it started working 😁 with alarm straight
away 😅 Thank you so much for all your help with this.

Best Regards
Daria

**Reply by Navid200** [reply]
I didn't do anything.  I was waiting for you to post a screenshot of the status page so that I could possibly see what could be the cause of the problem.

I'm glad it is working.
